corehq/apps/api/tests/user_resources.py

```diff
@@ -35,6 +38,8 @@
 from corehq.util.test_utils import generate_cases
 
 from .utils import APIResourceTest
+from ..resources.v0_5 import CommCareUserResource
+from ...domain.shortcuts import create_domain
```

Comment:
I'm not sure if this is best practice by I think relative imports should be restricted to the same app.